code/modules/antagonists/changeling/powers/tiny_prick.dm

```diff
@@ -285,3 +296,80 @@
 	if(target.reagents)
 		target.reagents.add_reagent(/datum/reagent/consumable/frostoil, 30)
 	return TRUE
+
+/datum/action/changeling/sting/fake_changeling
+	name = "False Changeling Sting"
+	desc = "We silently sting our victim and inject them with a fleshy mass to make them appear as us with an armblade while we take on their identity."
+	helptext = "Makes the victim appears like you while you take on their appearance, the disguise is just a fake outer flesh which will decay after a while. \
+	Gained once you obtained both false arm blade and transformation sting."
+	button_icon_state = "false_ling"
+	chemical_cost = 25
+	dna_cost = 0
+	prereq_ability = list(/datum/action/changeling/sting/transformation,
+	/datum/action/changeling/sting/false_armblade
+	)
+
+/datum/action/changeling/sting/fake_changeling/sting_action(mob/living/user, mob/living/target)
+	. = ..()
+	if(!ishuman(user))
+		return
+	if(!ishuman(target))
+		return
+
+	var/mob/living/carbon/human/changeling = user
+	var/mob/living/carbon/human/victim = target
+
+	var/datum/dna/our_dna = changeling.dna
+	var/datum/dna/their_dna = victim.dna
+
+	victim.updateappearance(mutcolor_update = TRUE ,target_dna = our_dna)
+	changeling.updateappearance(mutcolor_update = TRUE ,target_dna = their_dna)
+
+	var/obj/item/melee/arm_blade/false/blade = new(target,1)
+	target.put_in_hands(blade)
+	target.visible_message(span_warning("A grotesque blade forms around [target.name]\'s arm!"), span_userdanger("Your arm twists and mutates, transforming into a horrific monstrosity!"), span_hear("You hear organic matter ripping and tearing!"))
+	playsound(target, 'sound/effects/blob/blobattack.ogg', 30, TRUE)
+
+	addtimer(CALLBACK(src, PROC_REF(remove_effect), target, blade), 30 SECONDS)
+	return TRUE
+
+/datum/action/changeling/sting/fake_changeling/proc/remove_effect(mob/living/carbon/human/target, obj/item/melee/arm_blade/false/blade)
+	playsound(target, 'sound/effects/blob/blobattack.ogg', 30, TRUE)
+	target.visible_message(span_warning("With a sickening crunch, [target] reforms [target.p_their()] [blade.name] into an arm!"),
+	span_warning("[blade] reforms back to normal."), span_italics("You hear organic matter ripping and tearing!"))
+
+	qdel(blade)
+	target.update_held_items()
+	target.updateappearance(mutcolor_update = TRUE)
+
+/datum/action/changeling/sting/false_revival
+	name = "False Revival"
+	desc = "We inject a significant amount of ourselves to jerry rig the dead body back to life, immitading our revival stasis"
```

Comment:
"Jerry rig" is a strange word to use in the context of a changeling, not very thematic. 

Also typo: `immitading` -> `imitating`